src/test/java/com/google/api/codegen/testdata/discoveries/nodejs/nodejs_storage.v1.json.baseline

```diff
@@ -1353,6 +1353,13 @@ google.auth.getApplicationDefault(function(err, authClient) {
       // TODO: Add desired properties of the request body.
     },
 
+    // TODO: Use `alt: 'media'` to download media content.
```

Comment:
Wording change seems fine to me, but I suggest keeping my indentation, which is what the code would be at if used. Changed.